lambdas/data-migration2/src/granulesAndFiles.ts

```diff
@@ -0,0 +1,199 @@
+import Knex from 'knex';
+
+import DynamoDbSearchQueue from '@cumulus/aws-client/DynamoDbSearchQueue';
+import { ApiFile } from '@cumulus/types/api/files';
+import {
+  CollectionPgModel,
+  GranulePgModel,
+  FilePgModel,
+  PostgresFile,
+  PostgresGranuleRecord,
+  translateApiGranuleToPostgresGranule,
+} from '@cumulus/db';
+import { envUtils } from '@cumulus/common';
+import Logger from '@cumulus/logger';
+
+import { RecordAlreadyMigrated } from '@cumulus/errors';
+import { MigrationSummary } from './types';
+
+const logger = new Logger({ sender: '@cumulus/data-migration/granules' });
+const Manager = require('@cumulus/api/models/base');
+const schemas = require('@cumulus/api/models/schemas');
+const { getBucket, getKey } = require('@cumulus/api/lib/FileUtils');
+const { deconstructCollectionId } = require('@cumulus/api/lib/utils');
+
+export interface GranulesAndFilesMigrationSummary {
+  granulesSummary: MigrationSummary,
+  filesSummary: MigrationSummary,
+}
+
+/**
+ * Migrate granules record from Dynamo to RDS.
+ *
+ * @param {AWS.DynamoDB.DocumentClient.AttributeMap} record
+ *   Record from DynamoDB
+ * @param {Knex} knex - Knex client for writing to RDS database
```

Comment:
In reality this parameter is now a Knex transaction, so we should update the JSdoc

Reply:
[`393105e` (#2051)](https://github.com/nasa/cumulus/pull/2051/commits/393105ed535e7d30036ccdb188bee42b233dd4f5)

Shouldn't I update the type too?